tests/posix/message_passing/src/main.c

```diff
@@ -59,13 +57,12 @@ static void *receiver_thread(void *p1)
 	zassert_false(strcmp(rec_data, send_data), "Error in data reception. exp: %s act: %s",
 		      send_data, rec_data);
 	usleep(USEC_PER_MSEC);
-	zassert_false(mq_close(mqd),
-		      "unable to close message queue descriptor.");
+	zassert_false(mq_close(mqd), "unable to close message queue descriptor.");
 	pthread_exit(p1);
 	return NULL;
 }
 
-ZTEST(mqueue, test_mqueue)
+ZTEST(message_passing, test_message_passing)
```

Comment:
Only the testsuite name should change here, not the test name, here and in other tests